### DISCUSSION TITLE
Remove the SAP prefix from "Systems", and everywhere else.

### DISCUSSION THREAD

**Discussion by stefanotorresi** [trento-project/trento#86] [General]
Since the entire application is SAP focused, [it was suggested](https://github.com/trento-project/trento/pull/34#issuecomment-828358548) by @dirkmueller to remove the `SAP` prefix from the `SAP Systems` entry in the sidebar menu (which I implemented in #84).

Now that I think about it, though, I would like to know if we are all onboard with this change, especially @lee-martin.

More in general, shall we remove the "SAP" word from every core logical entity of the project? I'd like to achieve consensus on this.

I personally long for consistency, and using the prefix just for "Systems" doesn't feel consistent, as Dirk pointed out.

**Comment by dirkmueller** [comment]
currently its documented the opposite (but the implementation doesn't match documentation) and LAMA does it the other way around : 
https://github.com/trento-project/trento/blob/main/docs/trento-architecture.md#nomenclature

![image](https://user-images.githubusercontent.com/1029152/121678540-97d3d400-cab7-11eb-829a-1daf97315c20.png)

**Comment by lee-martin** [comment]
I agree with @stefanotorresi on consistency and would like to *keep* the SAP prefix on situations where it applies.

Since this is a non-SAP application, I do not find it inherently obvious to users that "Systems" means an SAP System, or SAP Landscape. The generic term System and Landscape is used so often to mean just the server/OS and landscapes describe a collection thereof - and neither of those definitions fir Trento.  I was wondering where the SAP prefix had disappeared to in 0.1.0.

Even LaMa used the "SAP" prefix when it wants to be specific, like "SAP Databases"

We may in parallel also want to consider not having things like hosts and cluster on the top level menu as well (Stefano and I were involved some recent discussion in this area).  Thats another topic though.

Still, I vote for having the prefix, like SAP Systems and SAP Landscapes. (not hosts though)
